spec/index.bs

```diff
@@ -15118,13 +15118,13 @@ The main rendering algorithm:
             [$process fragment$](|rasterPoint|, |descriptor|, |state|)
             for each |rasterPoint| in |rasterizationList|.
 
-        1. **Process depth/stencil**. See [[#depth-stencil-processing]]
+        1. **Process depth/stencil**. See [[#depth-stencil-processing]].
```

Comment:
What's a bit weird is that processing fragments seems like it should include everything that's per-fragment, so from early-Z to output merging:

 - Process fragments:
     - for each fragment, in parallel (also needs to mention quads?)
       - Early-Z / stencil, potentially continue;
       - Run FS, potentially continue;
       - Late Z / stencil, potentially continue;
       - Output merging.

Reply:
Thinking about it more that's not possible because per-sample operations vs. per-pixel. That's mildly annoying.